docs/content/integrations/powerbi.mdx

```diff
@@ -86,26 +82,17 @@ resource = PowerBIWorkspace(
 )
 
 
-# A translator class lets us customize properties of the built
-# Power BI assets, such as the owners or asset key
-class MyCustomPowerBITranslator(DagsterPowerBITranslator):
-    def get_report_spec(self, data: PowerBIContentData) -> AssetSpec:
-        # We add a team owner tag to all reports
-        return super().get_report_spec(data)._replace(owners=["my_team"])
-
-    def get_semantic_model_spec(self, data: PowerBIContentData) -> AssetSpec:
-        return super().get_semantic_model_spec(data)._replace(owners=["my_team"])
-
-    def get_dashboard_spec(self, data: PowerBIContentData) -> AssetSpec:
-        return super().get_dashboard_spec(data)._replace(owners=["my_team"])
+def customize_powerbi_asset(spec: AssetSpec) -> AssetSpec:
+    # We prefix all dashboard asset keys with "powerbi-dashboard" for
+    # organizational purposes
+    key = spec.key
+    if spec.metadata["dagster-powerbi/asset-type"] == "dashboard":
+        key = key.with_prefix("powerbi-dashboard")
 
-    def get_dashboard_asset_key(self, data: PowerBIContentData) -> AssetKey:
-        # We prefix all dashboard asset keys with "powerbi" for organizational
-        # purposes
-        return super().get_dashboard_asset_key(data).with_prefix("powerbi")
+    return spec._replace(key=key, owners="my_team")
```

Comment:
This is an interesting decision point where we leak the implementation detail that we inherit from `NamedTuple` in a heavy way. We probably can't change things at this point, but worth nothing that we are telling users to use `_replace` when the might expect `dataclass` or `pydantic` variants of object copies.